app/styles/ui/_button.scss

```diff
@@ -83,3 +83,37 @@
   padding: 0 var(--spacing-half);
   height: 21px;
 }
+
+.button-group-item {
+  border-right-width: 0;
+  border-radius: 0;
+
+  &:first-child {
```

Comment:
Do you want to extract these to a class in case the group of buttons aren&rsquo;t the only thing inside the parent element?

Reply:
@j-f1 do you mean increase the specificity of this selector to only look at a specific element/clas? or use a class as opposed to `:first-child`?

Reply:
Use a class as opposed to `:first-child` — CSS unfortunately only lets you find the first of a given element type and doesn’t let you find the first of a given class.